src/mars_patcher/door_locks.py

```diff
@@ -361,3 +404,28 @@ def tile_exists() -> bool:
                         h_flip,
                         v_flip,
                     )
+
+
+def get_blank_minimap_tile_id(blank_tile_ids: set[int], is_item: bool) -> int | None:
+    """Finds a usable tile from the provided set of blank tile IDs. Item tiles require a blank
+    tile next to them. Non-item tiles can use any blank tile, but solitary tiles are preferred
+    to save more tiles for item tiles."""
+    valid_tile_id: int | None = None
+    for tile_id in blank_tile_ids:
+        if is_item:
+            # Item tiles require a blank tile next to them for the obtained item tile
+            if tile_id + 1 in blank_tile_ids:
+                valid_tile_id = tile_id
+                break
+        else:
+            # Prefer solitary blank tiles for non-item tiles
+            if tile_id + 1 not in blank_tile_ids:
+                valid_tile_id = tile_id
+                break
+            elif valid_tile_id is None:
+                valid_tile_id = tile_id
+    if valid_tile_id is not None:
```

Comment:
doing this seems kinda weird, since BLANK_TILE_IDS / TRANSPARENT_TILE_IDS are supposed to be constants.
If you're doing it like this, I'd rather have a copy of the variables where it's clearer that they'll be getting modified.